### PR TITLE
python3Packages.meshcat: init at 0.3.2

### PR DESCRIPTION
**Pull request by wegank**
###### Description of changes

MeshCat is a remotely-controllable 3D viewer, built on top of three.js.

https://github.com/rdeits/meshcat-python

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/meshcat/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    u-msgpack-python
+    numpy
+    tornado
+    pyzmq
+    pyngrok
+    pillow
+  ];
+
+  # requires a running server
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/meshcat/default.nix:37]: pythonImportsCheck is missing

File: pkgs/development/python-modules/meshcat/default.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    ipython
+    u-msgpack-python
+    numpy
+    tornado
+    pyzmq
+    pyngrok
+    pillow
+  ];
+
+  # requires a running server
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/meshcat/default.nix:36]: What kind of server?

File: pkgs/development/python-modules/pyngrok/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-dws4Z4LzgkkOGTS5LZn/MU8ZKr70n4PWocezsDhxeT4=";
+  };
+
+  propagatedBuildInputs = [
+    pyyaml
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyngrok/default.nix:19]: checkPhase and pythonImportsCheck are missing.

File: pkgs/development/python-modules/meshcat/default.nix (hunk 1 above)
Comment (wegank) [reply pkgs/development/python-modules/meshcat/default.nix:39]: Added.

File: pkgs/development/python-modules/meshcat/default.nix (hunk 2 above)
Comment (wegank) [reply pkgs/development/python-modules/meshcat/default.nix]: In fact, a [MeshCat viewer](https://github.com/rdeits/meshcat/#meshcat).

**Comment by bcdarwin** [comment]
`nixpkgs-review` happy, x64-NixOS:

```
4 packages built:
python310Packages.meshcat python310Packages.pyngrok python39Packages.meshcat python39Packages.pyngrok
```

**Review by bcdarwin — APPROVED** [review]

**Pull request merged**